src/content/en/2019/accessibility.md

```diff
@@ -245,40 +247,43 @@ The relative popularity of the [dialog role](https://developer.mozilla.org/en-US
+In our sample, a lot of websites use buttons or links. From those, only around a quarter (24.39%) include textual labels with these controls. If a control is not labeled a screen reader user might read something generic, such as the word "button" instead of a meaningful word like "Search".
 
-Buttons and links are almost always included in the tab order and thus have extremely high visibility. Navigating through a website using the tab key is one of the primary ways through which users who use only the keyboard explore your website. So, a user is sure to encounter your unlabeled buttons and links if they are moving through your website using the tab key.
+Buttons and links are almost always included in the tab order and thus have extremely high visibility. Navigating through a website using the tab key is one of the primary ways through which users who use only the keyboard explore your website. So a user is sure to encounter your unlabeled buttons and links if they are moving through your website using the tab key.
 
 ### Accessibility of Form Controls
 
-Filling out forms is a task many of us do every single day. Whether we’re shopping, booking travel, or applying for a job, forms are the main way users share information with web pages. Because of this, ensuring your forms are accessible is incredibly important. The simplest means of accomplishing this is by providing labels (via the [`<label>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label), [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) or [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute)) for each of your inputs. Sadly, only **22.33% of pages **provide labels for all their form inputs, meaning 4 out of every 5 pages have forms that may be next to impossible to fill out.
+Filling out forms is a task many of us do every single day. Whether we’re shopping, booking travel, or applying for a job, forms are the main way users share information with web pages. Because of this, ensuring your forms are accessible is incredibly important. The simplest means of accomplishing this is by providing labels (via the [`<label>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label), [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) or [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute)) for each of your inputs. Sadly, only 22.33% of pages provide labels for all their form inputs, meaning 4 out of every 5 pages have forms that may be next to impossible to fill out.
```

Comment:
```suggestion
Filling out forms is a task many of us do every single day. Whether we’re shopping, booking travel, or applying for a job, forms are the main way users share information with web pages. Because of this, ensuring your forms are accessible is incredibly important. The simplest means of accomplishing this is by providing labels for each of your inputs. This can be accomplished via the [`<label>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label), [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) attribute, or [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) attribute. Sadly, only 22.33% of pages provide labels for all their form inputs, meaning 4 out of every 5 pages have forms that may be next to impossible to fill out.
```

Reply:
> may be next to impossible to fill out

Is this overly dramatic? Or is it true that without labels it's completely inaccessible?

Reply:
It may not be completely impossible, but it becomes much more difficult. You can hunt around for `<div>`s that precede the inputs and piece together what inputs are for what. But i know a lot of sites also replace labels with the `placeholder` attribute which is still not widely supported on assistive technology.

I think it's fair to at least say it'd be exceptionally difficult.

@kleinab @ljme what do you both think?